persistence/pagination/Paginator_test.go

```diff
@@ -0,0 +1,20 @@
+package pagination_test
+
+import (
+	"testing"
+
+	"github.com/ditrit/badaas/persistence/pagination"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test(t *testing.T) {
+	paginator := pagination.NewPaginator(uint(0), uint(12))
+	assert.NotNil(t, paginator)
+	assert.GreaterOrEqual(t, paginator.Offset(), uint(1))
```

Comment:
why GreatorOrEQual insteal of equal ?

Reply:
Because we provided uint(0) as an offset in the contructor and we want to get uint(1) in response